akka-persistence-typed/src/main/scala/akka/persistence/typed/internal/Running.scala

```diff
@@ -97,6 +109,48 @@ private[akka] object Running {
   import InternalProtocol._
   import Running.RunningState
 
+  import scala.concurrent.duration._
+  import akka.actor.typed.scaladsl.AskPattern._
+
+  private val log = setup.log
+
+  def startReplicationStream(state: RunningState[S]): Unit = {
+    setup.activeActive.map { aa =>
+      val query = PersistenceQuery(setup.context.system)
```

Comment:
Did we already have persistence query as a dependency for akka-persistence-typed or how is this possible to access?

Reply:
We had it as a test dependency. I changed it to a prod dependency. 
I think this is okay as they are typically used together.
As we have persistence-typed this avoided a cyclic dependency that we would have had for untyped